### PR TITLE
BOSA21Q1-799 Add the "placeholder" attribute to the text-area for create proposal form

### PR DESCRIPTION
**Pull request by sergei-krylov**
Add the "placeholder" attribute to the text-area for create proposal form.
Remove aria-describedby and replace it with an “aria-label” with the same content as the “placeholder".

### CODE REVIEW

File: lib/extends/decidim-proposals/app/helpers/decidim/proposals/application_helper.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        class: "js-hashtags",
+        hashtaggable: true,
+        value: form_presenter.body(extras: false).strip,
+        "aria-describedby" => ''
+      }
+
+      placeholder_tkey = "decidim.proposals.proposals.proposal.body_placeholder"
+      placeholder_text = I18n.exists?(placeholder_tkey) && t(placeholder_tkey).present? ? t(placeholder_tkey) : ''
+      options.merge!(
+        placeholder: I18n.exists?(placeholder_tkey) && t(placeholder_tkey).present? ? t(placeholder_tkey) : '',
```
Comment (ThomasCollignon) [inline lib/extends/decidim-proposals/app/helpers/decidim/proposals/application_helper.rb:21]: Why not use `placeholder_text` here ?

Could quickly tell me the diff between `I18n.exists?(x) `and `t(x).present?` ?
I find it strange that `t(x).present?` isn't used anywhere else in the code.

**Comment by ThomasCollignon** [comment]
If you don't provide a default translation in `[locale].yml`, in the admin config it will be the only line for which there is a blank value in column _Original term_.

File: lib/extends/decidim-proposals/app/helpers/decidim/proposals/application_helper.rb (hunk 1 above)
Comment (sergei-krylov) [reply lib/extends/decidim-proposals/app/helpers/decidim/proposals/application_helper.rb]: Hah, missed that during the refactoring at the end. It should be a `placeholder_text` you're right.
Updated!

Comment (sergei-krylov) [reply lib/extends/decidim-proposals/app/helpers/decidim/proposals/application_helper.rb]: > Could quickly tell me the diff between I18n.exists?(x) and t(x).present? ?

If the key doesn't exist in translations the `t(key)` (same as `I18n.t(key)`) will fail with "missing translation" exception. The `I18n.exists?(key)` is checking if the key exists and doesn't raise exception.

**Comment by sergei-krylov** [comment]
> If you don't provide a default translation in [locale].yml, in the admin config it will be the only line for which there is a blank value in column Original term.

Right, and that's not an issue. The "default" value is just blank, in case we don't need to display any placeholder.

**Review by ThomasCollignon — APPROVED** [review]

**Review by simon-lejeune — APPROVED** [review]

**Pull request merged**